src/Raven.Server/Documents/DatabasesLandlord.cs

```diff
@@ -582,7 +582,7 @@ public Task<DocumentDatabase> TryGetOrCreateResourceStore(StringSegment database
             try
             {
                 if (_wakeupTimers.TryRemove(databaseName.Value, out var timer))
-                    timer.Dispose();
+                    timer.Value.Dispose();
```

Comment:
do we want to check if it was materialized?

Reply:
done